tests/mock_crust/churn.rs

```diff
@@ -90,6 +92,44 @@ fn did_receive_get_success(node: &TestNode,
     }
 }
 
+fn quorum_before(nodes: &[TestNode],
+                 src: Authority,
+                 dst: Authority,
+                 data_id: DataIdentifier,
+                 message_id: MessageId)
+                 -> usize {
+    let group = nodes.iter()
+        .filter(|n| n.routing_table().is_recipient(&src.to_destination()))
+        .collect_vec();
+
+    for node in &group {
+        let _ = node.inner.send_get_request(src, dst, data_id, message_id);
+    }
+
+    (group.len() * QUORUM - 1) / 100 + 1
+}
```

Comment:
The name `quorum_before` doesn't suggest that it also sends a request. I'd just put lines 105-107 into the tests themselves, and only calculate the quorum here.